app/src/main/res/xml/settings.xml

```diff
@@ -25,60 +25,34 @@
 			android:entryValues="@array/settings_interval_values"
 			android:inputType="number"
 			android:key="refresh_interval"
-			android:title="@string/settings_refresh_interval"/>
+			android:title="@string/settings_refresh_interval" />
 
 		<CheckBoxPreference
 			android:layout_width="wrap_content"
 			android:layout_height="wrap_content"
 			android:defaultValue="false"
 			android:key="refresh_wifi_only"
 			android:summary="@string/settings_refresh_wifi_only_description"
-			android:title="@string/settings_refresh_wifi_only"/>
+			android:title="@string/settings_refresh_wifi_only" />
 	</PreferenceCategory>
 
 	<PreferenceCategory
 		android:layout_width="wrap_content"
 		android:layout_height="wrap_content"
-		android:title="@string/settings_category_content_presentation">
+		android:title="@string/settings_category_filter">
 
-		<net.frju.flym.ui.views.AutoSummaryListPreference
-			android:layout_width="wrap_content"
-			android:layout_height="wrap_content"
-			android:defaultValue="4"
-			android:entries="@array/settings_keep_times"
-			android:entryValues="@array/settings_keep_time_values"
-			android:inputType="number"
-			android:key="keep_time"
-			android:title="@string/settings_keep_time"/>
-
-		<net.frju.flym.ui.views.AutoSummaryListPreference
-			android:layout_width="wrap_content"
-			android:layout_height="wrap_content"
-			android:defaultValue="0"
-			android:entries="@array/settings_font_sizes"
-			android:entryValues="@array/settings_font_size_values"
-			android:inputType="number"
-			android:key="font_size"
-			android:title="@string/settings_font_size"/>
 
 		<CheckBoxPreference
 			android:layout_width="wrap_content"
 			android:layout_height="wrap_content"
 			android:defaultValue="true"
-			android:key="display_images"
-			android:summary="@string/settings_display_images_description"
-			android:title="@string/settings_display_images"/>
+			android:key="remove_duplicates"
```

Comment:
I'm not so sure about this. Maybe some users want to know if there are duplicates for whatever reason. A function which automatically deletes something should always be optional in my opinion. Otherwise, users might complain about it or see it as a bug.

Reply:
I understand your point, and it's true that some people may be surprised, but on the other side:
- I don't see any use cases where you would like to keep the duplicates (unless the feature does have some downsides, but at first sight it is not the case)
- I don't think most people will actually ever notice the difference

Hence I believe this is certainly an uneeded complication. However, if you still believe I'm wrong and you want to have this option anyway, I'm willing to accept it.